src/test/java/stacksandqueues/QueueTest.java

```diff
@@ -0,0 +1,65 @@
+package stacksandqueues;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class QueueTest {
+    //enqueue multiple items
+    @Test
+    public void testEnqueue() {
+        Queue testQueue = new Queue();
+        testQueue.enqueue(1);
+        testQueue.enqueue(2);
+        testQueue.enqueue(3);
+
```

Comment:
You could also have assert statements between each enqueue to verify that it is doing what is expected. As well as dequeue something then enqueue it to ensure that it's right. 

What happens when DQed to null then enqueued?